.github/workflows/release_upload_bundle.yml

```diff
@@ -0,0 +1,102 @@
+name: Release - Upload Bundle to Play Store
+
+on:
+  workflow_dispatch:
+
+env:
+  ASANA_PAT: ${{ secrets.GH_ASANA_SECRET }}
+  GOOGLE_APPLICATION_CREDENTIALS: '#{ENV["HOME"]}/jenkins_static/com.duckduckgo.mobile.android/ddg-upload-firebase.json'
+
+jobs:
+  create-tag:
+    name: Generate and upload nightly bundle to Play Store Internal track
+    runs-on: macos-latest
```

Comment:
Is there a specific reason we need a macOS runner for this workflow? I understand it’s not intended to be executed often, but it still feels wasteful if there are no blockers to using Ubuntu, which is significantly cheaper, IIRC.

Reply:
You are right, there’s no need. Moved it all to ubuntu-latest